owncloudApp/src/main/res/layout/preview_audio_fragment.xml

```diff
@@ -29,32 +29,51 @@
 
     <FrameLayout
         android:id="@+id/visual_area"
-        android:layout_width="match_parent"
+        android:layout_width="0dp"
         android:layout_height="0dp"
-        android:layout_alignParentTop="true"
         android:layout_above="@+id/media_controller"
-        >
+        android:layout_alignParentTop="true"
+        android:layout_centerHorizontal="true"
+        android:padding="40dp"
+        app:layout_constraintDimensionRatio="H,1:1"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toTopOf="@id/media_controller"
+        android:elevation="16dp"
+        android:outlineProvider="paddedBounds"
+        android:clipToPadding="false">
 
         <ImageView
             android:id="@+id/image_preview"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
-            android:layout_margin="@dimen/standard_margin"
-            android:layout_gravity="center"
             android:contentDescription="@string/preview_image_description"
-            android:src="@drawable/logo" />
+            android:scaleType="centerCrop"
+            android:src="@drawable/ic_place_holder_music_cover_art"
+            app:layout_constraintBottom_toBottomOf="parent" />
```

Comment:
The parent is not a Constraint layout, we don't need this constraint here, am I right?